kernel/src/main/java/org/kframework/kompile/Kompile.java

```diff
@@ -105,7 +105,7 @@ public Kompile(KompileOptions kompileOptions, FileUtil files, KExceptionManager
         this.sw = sw;
 
         if (kompileOptions.backend.equals("ocaml")) {
-            kem.registerCriticalWarning("The OCaml backend is in the process of being deprecated (final date May 31, 2020). Please switch to the LLVM backend.");
+            kem.registerCriticalWarning(ExceptionType.DEPRECATED_BACKEND, "The OCaml backend is in the process of being deprecated (final date May 31, 2020). Please switch to the LLVM backend.");
         }
```

Comment:
Is there much use in distinguishing `DEPRECATED_BACKEND` from `FUTURE_ERROR`? Seems to me that `DEPRECATED_BACKEND` will only be used twice, and equally well could be just thrown into "these are features that we will not support in the future.